app/components/post_textbox/post_textbox.js

```diff
@@ -474,7 +474,7 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
     return StyleSheet.create({
         buttonContainer: {
             height: 36,
-            width: 36,
+            width: 45,
```

Comment:
concerns on how does this look on IOS and Android, and by changing this, don't we need to change the padding on the input?